ext/simplexml/simplexml.stub.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/** @return SimpleXMLElement */
+function simplexml_load_file(string $filename, string $class_name = 'SimpleXMLElement', int $options = 0, string $ns = null, bool $is_prefix = false) {}
```

Comment:
```suggestion
function simplexml_load_file(string $filename, ?string $class_name = SimpleXMLElement::class, int $options = 0, string $ns = '', bool $is_prefix = false) {}
```

$class_name is nullable, $ns isn't.